datashuttle/datashuttle.py

```diff
@@ -553,21 +562,30 @@ def download_specific_folder_or_file(
             transfer was taking place, but no files will be moved. Useful
             to check which files will be moved on data transfer.
         """
+        if isinstance(filepath, str):
+            filepath = Path(filepath)
+
         self._start_log(
             "download-specific-folder-or-file", local_vars=locals()
         )
 
-        self._display_top_level_folder()
+        top_level_folder = self._get_top_level_folder_from_specific_filepath(
+            filepath
+        )
```

Comment:
In the docstring of `download_specific_folder_or_file` and `upload_specific_folder_or_file` you mention:
> filepath :
>   a string containing the filepath to move,
>   relative to the project folder "rawdata"
>   or "derivatives" path (depending on which is currently
>   set). Alternatively, the entire path is accepted.

This is no longer true, right? The docstring needs to be updated according to the new behaviour.

Reply:
Nevermind, I saw you have fixed this as part of [another PR](https://github.com/neuroinformatics-unit/datashuttle/pull/348).